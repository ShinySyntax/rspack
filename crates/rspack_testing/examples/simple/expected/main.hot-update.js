self['hotUpdate']('main', {
"./a.js": function (module, exports, __webpack_require__) {
console.log('b');
},
"./index.js": function (module, exports, __webpack_require__) {
__webpack_require__("./a.js");
__webpack_require__("./d.js");
},

});