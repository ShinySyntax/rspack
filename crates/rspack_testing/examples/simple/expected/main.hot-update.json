{"c":["main"],"r":[],"m":["./c.js"]}